### PR TITLE
Export parsed tag body as an output

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ outputs:
   filename:
     description: "Copy of inputs.filename, to re-use it for other jobs"
     value: ${{ inputs.filename }}
+  body:
+    description: "Tag details without the Subject"
```
Comment (allisonkarlitskaya) [inline action.yml]: This capital S is funny.

File: action.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ outputs:
...
+    value: ${{ steps.publis.outputs.body }}
```
Comment (allisonkarlitskaya) [inline action.yml]: `publis`?

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (martinpitt) [reply action.yml]: Fixed.

File: action.yml (hunk 2 above)
Comment (martinpitt) [reply action.yml]: Ouch! Fixed.

**Review by allisonkarlitskaya — APPROVED** [review]
Thanks.  Looking forward to using this.

**Pull request merged**